CHANGELOG.md

```diff
@@ -1,15 +1,137 @@
+by [@rsnara](https://github.com/rsnara))
+- Add `accessibilityHint` for iOS ([253b29d](https://github.com/facebook/react-native/commit/253b29d) by [@draperunner](https://github.com/draperunner))
+
+
+### Changes: existing functionality that is now different
+
+- Bump minimum Node version to 8.3 (#20236) ([e64e13f](https://github.com/facebook/react-native/commit/e64e13f) by [@hramos](https://github.com/hramos))
+- Updated React ([70913a4](https://github.com/facebook/react-native/commit/70913a4) and [b7bb25f](https://github.com/facebook/react-native/commit/b7bb25f) by [@acdlite](https://github.com/acdlite)) and [@hramos](https://github.com/hramos))
+- Upgrade Flow to v0.76.0 ([eac34e3](https://github.com/facebook/react-native/commit/eac34e3) by [@gabelevi](https://github.com/gabelevi))
+- Upgrade jest to 23.4.1 ([51cf9eb](https://github.com/facebook/react-native/commit/51cf9eb) by [@rafeca](https://github.com/rafeca))
+- Upgrade babel-eslint to v9.0.0-beta.2 with better support for Flow ([abf1188](https://github.com/facebook/react-native/commit/abf1188) by [@rubennorte](https://github.com/rubennorte))
+- Upgrade ESLint to 5.1.0 ([0f2f0ca](https://github.com/facebook/react-native/commit/0f2f0ca) by [@rubennorte](https://github.com/rubennorte))
+- Upgrade Babel to v7.0.0-beta.54 ([b9d1c83](https://github.com/facebook/react-native/commit/b9d1c83) and [724c749](https://github.com/facebook/react-native/commit/724c749) by [@rubennorte](https://github.com/rubennorte))
+- Metro is now at v0.43.6 ([169d683](https://github.com/facebook/react-native/commit/169d683), [bda84a3](https://github.com/facebook/react-native/commit/bda84a3), and [5288656](https://github.com/facebook/react-native/commit/5288656) by [@CompuIves](https://github.com/CompuIves) and [@rafeca](https://github.com/rafeca))
```

Comment:
metro, babel and babel-eslint and flow will need to be revised once we cherry pick and merge the necessary commits

Reply:
Will watch for that.